spec/models/meter_reading_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe MeterReading, type: :model do
+  let!(:meter_reading) { create(:meter_reading) }
+  let(:value) { meter_reading.value }
+  let(:date) { meter_reading.date }
+
+  context "when valid Factory" do
```

Comment:
this test test should be covered by unified factory spec so it's useless now just takes more time for spec to run

Reply:
done